doc/under_sampling.rst

```diff
@@ -365,12 +365,32 @@ and the output a 3 nearest neighbors classifier. The class can be used as::
 
 .. _instance_hardness_threshold:
 
+Additional undersampling techniques
```

Comment:
The user guide is divided into controlled undersampling (RUS and nearmiss) and cleaning (all the others). I felt we needed another headline for IHT